Technical Charter.md

```diff
@@ -1,242 +0,0 @@
-
-    1.  Collaborators acknowledge that the copyright in all new contributions
-        will be retained by the copyright holder as independent works of
-        authorship and that no contributor or copyright holder will be required
-        to assign copyrights to the Project.
-
-    2.  Except as described in Section 7.c., all contributions to the Project
-        are subject to the following:
-
-        1.  All new inbound code contributions to the Project must be made using
-            the Apache License, Version 2.0, available at
-            [https://www.apache.org/licenses/LICENSE-2.0](https://www.apache.org/licenses/LICENSE-2.0) (the “Project
-            License”).
-
-        2.  All new inbound code contributions must also be accompanied by a
-            Developer Certificate of Origin ([http://developercertificate.org>](http://developercertificate.org))
-            sign-off in the source code system that is submitted through a
-            TSC-approved contribution process which will bind the authorized
-            contributor and, if not self-employed, their employer to the
-            applicable license;
-
-        3.  All outbound code will be made available under the Project License.
-
-        4.  Documentation will be received and made available by the Project
-            under the Creative Commons Attribution 4.0 International License
-            (available at [http://creativecommons.org/licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/)).
-
-        5.  To the extent a contribution includes or consists of data, any
-            rights in such data shall be made available under the
-            [CDLA-Permissive 1.0 License](https://cdla.io/permissive-1-0/).
-
-        6.  The Project may seek to integrate and contribute back to other open
-            source projects (“Upstream Projects”). In such cases, the Project
-            will conform to all license requirements of the Upstream Projects,
-            including dependencies, leveraged by the Project. Upstream Project
-            code contributions not stored within the Project’s main code
-            repository will comply with the contribution process and license
-            terms for the applicable Upstream Project.
-
-    3.  The TSC may approve the use of an alternative license or licenses for
-        inbound or outbound contributions on an exception basis. To request an
-        exception, please describe the contribution, the alternative open source
-        license(s), and the justification for using an alternative open source
-        license for the Project. License exceptions must be approved by a
-        two-thirds vote of the entire TSC.
-
-    4.  Contributed files should contain license information, such as SPDX short
-        form identifiers, indicating the open source license or licenses
-        pertaining to the file.
-
-8.  Amendments
-
-    1.  This charter may be amended by a two-thirds vote of the entire TSC and

```

Comment:
Do we want to keep 2/3 for amending the Charter, or use 50% majority like we'd use for amending the Governance doc?

I figure 50% is fine because Charter changes also go in front of the CPC

Reply:
With a TSC of 4 people you need 3 out of 4 to have a majority, which is 75%, so it comes to the same thing no?

Reply:
You're right - I misunderstood. So yeah I think it comes out to be the same thing